internal/recert/recert.go

```diff
@@ -88,7 +62,23 @@ func CreateRecertConfigFile(clusterData *clusterinfo.ClusterInfo,
 	}
 	config.UseCertRules = []string{filepath.Join(certsDir, "admin-kubeconfig-client-ca.crt")}
 
-	return utils.WriteToFile(config, filepath.Join(clusterConfigPath, recertConfigFile))
+	return utils.WriteToFile(config, filepath.Join(recertConfigFolder, RecertConfigFile))
+}
+
+func CreateRecertConfigFileForSeedCreation(path string) error {
```

Comment:
I think we'd like to do a rename into a dummy name even during seed, to ensure rename works before imaging. Rename is not a simple operation, many things can fail it.

Reply:
Yeah :) in anycase we need seed ip so we will still need someinfo from seed. 
Lets move it to another PR, will need more testing